libs/sdk-react-native/ios/BreezSDKMapper.swift

```diff
@@ -455,7 +511,22 @@ class BreezSDKMapper {
             "message": messageSuccessActionData.message
         ]
     }
-    
+
+    static func dictionaryOf(receivePaymentResponse: ReceivePaymentResponse) -> [String: Any?] {
+        return [
+            "lnInvoice": dictionaryOf(lnInvoice: receivePaymentResponse.lnInvoice),
+            "openingFeeParams": dictionaryOf(params: receivePaymentResponse.openingFeeParams),
+            "openingFeeMsat": receivePaymentResponse.openingFeeMsat
+        ]
+    }
+
+    static func dictionaryOf(buyBitcoinResponse: BuyBitcoinResponse) -> [String: Any?] {
+        return [
+            "url": buyBitcoinResponse.url,
+            "openingFeeParams": dictionaryOf(params: buyBitcoinResponse.openingFeeParams)
```

Comment:
Will cause problems when openingFeeParams is nil

Reply:
Should be covered by above change to `dictionaryOf(openingFeeParams`